lib/iris/tests/test_basic_maths.py

```diff
@@ -653,73 +653,189 @@ def test_log10(self):
         self.assertCMLApproxData(e, ('analysis', 'log10.cml'), rtol=1e-6)
 
 
-class TestMaskedArrays(tests.IrisTest):
-    ops = (operator.add, operator.sub, operator.mul)
-    iops = (operator.iadd, operator.isub, operator.imul)
-    # This is to ensure compatibility for both Python2 (which uses div and
-    # idiv) and Python3 (which uses truediv and itruediv).
-    try:
-        ops = ops + (operator.div, )
-        iops = iops + (operator.idiv, )
-    except AttributeError:
-        ops = ops + (operator.truediv, )
-        iops = iops + (operator.itruediv, )
-
+class TestMathOperations(tests.IrisTest):
     def setUp(self):
-        self.data1 = ma.MaskedArray([[9, 9, 9], [8, 8, 8,]],
-                                    mask=[[0, 1, 0], [0, 0, 1]])
-        self.data2 = ma.MaskedArray([[3, 3, 3], [2, 2, 2,]],
-                                    mask=[[0, 1, 0], [0, 1, 1]])
-
-        self.cube1 = iris.cube.Cube(self.data1)
-        self.cube2 = iris.cube.Cube(self.data2)
+        self.data_1f = np.array([[9, 9, 9], [8, 8, 8]],
+                                dtype=np.float64)
```

Comment:
We actually weren't using the masked data at all, it's just what I inherited with these tests, and up until now it has been fine. But, weirdly, masked arrays screw up inplace operations on uint data, so I've removed the masking. The two tests in this module that actually were about masked data I've moved to a new class below.